ProjFS.Mac/PrjFSKext/VnodeCache.cpp

```diff
@@ -367,13 +433,17 @@ KEXT_STATIC bool TryFindVnodeIndex_Locked(
     /* out parameters */
```

Comment:
Food for thought, it would be nice to know this is only called under a shared lock.  Not part of this PR, but throwing it out there for readability in the future.

Reply:
This function is actually called under both shared and exclusive locks:

```
KEXT_STATIC bool TryInsertOrUpdateEntry_ExclusiveLocked(
    vnode_t _Nonnull vnode,
    uintptr_t vnodeHashIndex,
    uint32_t vnodeVid,
    bool forceRefreshEntry,
    VirtualizationRootHandle rootHandle)
{
    uintptr_t vnodeIndex;
    if (TryFindVnodeIndex_Locked(vnode, vnodeHashIndex, /*out*/ vnodeIndex))
    {
```

Do you think a longer name (e.g. _SharedOrExclusiveLocked) would be better?

Reply:
That would be helpful for me yes, but toally scope creep for this PR :-).  I can do it as a small separate PR if you're cool with it.

Reply:
> I can do it as a small separate PR if you're cool with it.

Sure if you don't mind that would be great 👍